app/models/user.rb

```diff
@@ -34,6 +34,8 @@ class User < ActiveRecord::Base
 
   belongs_to :system_role, class_name: "Role"
 
+  DEFAULT_PROFILE_PIC_URL = 'http://coursemology.s3.amazonaws.com/public/default_profile_pic.png'
```

Comment:
Hmm..actually I searched for a while and didn't find the answer, so normally where is the correct place ?